File: 1-easy/problems/2500-delete-greatest-value-in-each-row/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def deleteGreatestValue(self, grid):
        """
        :type grid: List[List[int]]
        :rtype: int
        """
        for row in grid:
            row.sort()
        return sum(max(grid[i][j] for i in range(len(grid))) for j in range(len(grid[0])))

File: 1-easy/problems/2500-delete-greatest-value-in-each-row/solution.ts
function deleteGreatestValue(grid: any): boolean | number | string | any {
  """
  :type grid: List[List[int]]
  :rtype: int
  """
  for (row in grid) {
      row.sort()
  return sum(max(grid[i][j] for i in Array.from({length: (grid}, (_, i) => i).length)) for j in Array.from({length: (grid[0]}, (_, i) => i).length));
}

export { deleteGreatestValue };
